src/components/Timelapse.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div id = "timelape">
+    <PlayButton v-if = "timelapseMode" />
+    <TimelapseBar v-if = "timelapseMode" />
+  </div>
+</template>
+
+<script>
+import TimelapseBar from "./TimelapseBar"
+import PlayButton from "./PlayButton"
+
+export default {
+  data () {
+    return {
+      timelapseMode: true //whether or not the map is in timelapse mode
```

Comment:
Why do we need `timelapseMode`? Is it related to any functionality we want to support in the future?

Reply:
That was intended to represent whether or not dates had been selected; if no dates have been selected, it makes sense to hide the timelapse bar and avoid confusion. If we're intending for the user to be able to run a timelapse at any point without having selected start and end dates and times, we can remove this statement and make the code cleaner.

Reply:
Okay let us keep this for now, once we finalize the dates UI we can decide what to do with this.